connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/EmailExceptionProcessor.java

```diff
@@ -1,30 +0,0 @@
-package com.redhat.cloud.notifications.connector.email;
-
-import com.redhat.cloud.notifications.connector.ExceptionProcessor;
-import io.quarkus.logging.Log;
-import jakarta.enterprise.context.ApplicationScoped;
-import org.apache.camel.Exchange;
-import org.apache.camel.http.base.HttpOperationFailedException;
-
-@ApplicationScoped
-public class EmailExceptionProcessor extends ExceptionProcessor {
```

Comment:
Because we are requesting internal resources (notifications-recipients-resolver or BOP) I think we should alway log errors using warn or error level.
On common part those errors will not be logged (because debug log should not be applied by default on prod)

Reply:
I made the log level configurable.